CONTRIBUTORS_QUICK_START.rst

```diff
@@ -387,8 +391,9 @@ Using Breeze
       </div>
 
 3. Setup mysql database in
-MySQL Workbench with Host ``127.0.0.1``, port ``23306``, user ``root`` and password
-   blank(leave empty), default schema ``airflow``.
+   PyCharm Database tool with Host ``127.0.0.1``, port ``23306``, user ``root`` and password
```

Comment:
This phrasing seems a bit off. It seems to presume that Pycharm is the default, which I'm not sure is accurate